src/persistency/PersistentStore.ts

```diff
@@ -13,11 +13,11 @@ export default class PersistentStore {
   }
 
   public createServiceNamespace(service: string, options: Record<string, any>): PersistentStore {
```

Comment:
These are two separate things. The user creates the storage they want Resuffle to use: memory, file or SQL. Services (and scripts) should create namespaces so that they don't accidentally step on each others' toes.

For example:

```
  private updateStore() {
    if (this.app) {
      this.store = this.app
        .getPersistentStore()
        .createServiceNamespace(this.constructor.name, this.account)
    }
  }
```

Reply:
I don't think we need to mandate a namespace, simplicity and flexibility please.

Reply:
Indeed that's what we agreed. I think you said "enforce it in the documentation" or something similar. This is why I built this helper function - it is not mandatory, but makes it easier for service developers to play nice together.